tests/user.test.js

```diff
@@ -0,0 +1,117 @@
+jest.mock('./../services/mailSender');
+
+const request = require('supertest');
+// const expect = require('expect');
+
+const {app} = require('./../app');
+const User = require("../models/user.model");
+
+
+
+var email = 'alice@example.com';
+var password = 'alicepasswd';
+var firstName = 'alice';
+var lastName = 'green';
+
+var userData = {
+    email, 
+    password,
+    firstName,
+    lastName
+};
+
+
+User.removeUser(email);
```

Comment:
This is for removing test user created previously, when the tests are run again.

Reply:
I know what It does, I am saying that that's async code that returns a promise, you should wait for that promise to resolve before executing any tests, (you can face race condition if you run it like this) use beforeAll (https://jestjs.io/docs/en/api#beforeallfn-timeout) resolve that in this the run your tests